### ISSUE TITLE
element->value() doesn't fill all characters

### ISSUE THREAD

**Issue by borekb** [giorgiosironi/phpunit-selenium#338]
We have occasional instances where `$element->value("Some value")` fills in only a portion of characters, e.g. "Some va" and then proceeds to the next `$element->value()`. This happens on the WordPress login form where we simply need to fill in username and password and click Submit.

Unfortunately, it happens irregularly and not very often but I thought I'd report it in case other users are hit by the same issue.

**Comment by zerkms** [comment]
Any chance it's some JS interferes with it?

**Comment by borekb** [comment]
I don't think so - maybe just the fact that some JS executed on the page generally slows the process down and the driver doesn't make it to fill in the whole value. But as I said, it's a really simple WordPress login form, like this one: http://demo.opensourcecms.com/wordpress/wp-login.php
